docs/exchanges.md

```diff
@@ -295,6 +295,53 @@ It's therefore required to pass the UID as well.
 !!! Warning "Necessary Verification"
     Bitmart requires Verification Lvl2 to successfully trade on the spot market through the API - even though trading via UI works just fine with just Lvl1 verification.
 
+## Hyperliquid
+
+!!! Tip "Stoploss on Exchange"
+    Hyperliquid supports `stoploss_on_exchange` and uses `stop-loss-limit` orders. It provides great advantages, so we recommend to benefit from it.
+
+Hyperliquid is a Decentralized Exchange (DEX). Decentralized exchanges work a bit different compared to normal exchanges. Instead of authenticating private API calls using an API key, private API calls need to be signed with the private key of your wallet. 
+This needs to be configured like this:
+
+```json
+"exchange": {
+    "name": "hyperliquid",
+    "walletAddress": "your_eth_wallet_address",
+    "privateKey": "your_private_key",
+    // ...
+}
+```
+
+walletAddress must be in hex format: `0x<40 hex characters>`, and can be easily copied from your wallet.
+
+privateKey also must be in hex format: `0x<64 hex characters>`. 
```

Comment:
are you sure this is true / necessary? 
I copy/pasted both walletAddress and PrivateKey directly out of metamask - and it did work fine (private key did NOT start with 0x).

Is there some documentation (ccxt or hyperliquid) to highlight that this would actually be necessary / improves performance or security ... ? 

I'd rather not recommend people install random dependencies - not considering the problem of pip package poisoning / typo squatting, which is especially prevalent in the crypto space.

Reply:
ah you're right, 0x is not necessary. ccxt strips the leading 0x part by taking only the last 64 characters (example: `signature = self.ecdsa(hash[-64:], privateKey[-64:], 'secp256k1', None)` ). I'll update that. I'll also add that it can be exported from metamask. Not sure if all wallets support that, for some you might need the mnemonic and re-generate the private key like in my example. I'll add another warning, let me know if you want to totally delete it.

Reply:
Hm, [here](https://github.com/ccxt/ccxt/blob/58074e6d03b96f0658ffb93dbce1db9d2f035bde/python/ccxt/base/exchange.py#L227) ccxt mentions it should be prefixed with 0x. It works without in this case, at this time, but maybe better to state that it should be prefixed with 0x to increase the chance that it will keep working in the future :)

Reply:
that works for me (0x won't hurt) - though i'm not sure the "convert from mnemonic" is really necessary - that seems to complicate things - it's more a "get a private key" topic - some wallets may not give you the mnemonic phrase (didn't test the ones hyperliquid supprots to be fair) - so the instruction may turn out to be rather missleading ...

Reply:
(done)